### PR TITLE
Implement bottom section of AboutScreen

### LINKED ISSUE #431: Implement Bottom Section of AboutScreen
**Idea Description**
Implement the bottom section of the AboutScreen, as divided into three parts in the design on [Figma](https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI). The task includes:
   - Implementing the design for the bottom section only.
   - Adding the design to the existing empty implementation in AboutFooterLinks.
   - No need to create any navigation behavior; simply replicating the design is sufficient.

**Reference images and links**
image

### PR DESCRIPTION
**Pull request by kokoakuma**
## Issue
- close #431

## Overview (Required)
- implement the bottom section of the AboutScreen as designed.
- delete explanation "Please Implement AboutScreen"
- modify the padding of AboutScreen

## Screenshot
Before | After | After
:--: | :--: | :--:
 |  |

### CODE REVIEW

File: feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutScreen.kt [hunk 1]
```diff
@@ -78,14 +87,12 @@ private fun AboutScreen(
         content = { padding ->
             LazyColumn(
                 Modifier
-                    .padding(padding),
+                    .padding(
```
Comment (kokoakuma) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutScreen.kt:90]: I modified the padding of AboutScreen a little
bacause unneccasary padding is passed to bottom like this.


If you know any other good idea, I'd like to hear them 🙏

File: feature/about/src/test/java/io/github/droidkaigi/confsched2023/AboutScreenTest.kt [hunk 2]
```diff
@@ -50,6 +50,9 @@ class AboutScreenTest {
             // Take a screenshot of the Others
             scrollAboutScreen()
             checkScreenCapture()
+            // Take a screenshot of the Footer Links
+            scrollAboutScreen()
+            checkScreenCapture()
```
Comment (takahirom) [inline feature/about/src/test/java/io/github/droidkaigi/confsched2023/AboutScreenTest.kt:55]: 👍

File: feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/component/AboutFooterLinksIcon.kt [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                contentDescription = "YouTube",
+                onClick = {},
+            )
+        }
+    }
+}
+
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+fun AboutFooterLinksIconDarkModePreview() {
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/component/AboutFooterLinksIcon.kt:58]: 👍

File: feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutScreen.kt [hunk 4]
```diff
@@ -45,6 +51,7 @@ const val AboutScreenTestTag = "AboutScreen"
 @Composable
 fun AboutScreen(
     onAboutItemClick: (AboutItem) -> Unit,
+    versionName: String? = versionName(LocalContext.current),
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutScreen.kt:54]: Can you use `remember{}` for this value for performance?

**Comment by takahirom** [comment]
Let me merge this as there are a lot of pull requests! Thanks for your contribution!

**Pull request merged**